crowbar.yml

```diff
@@ -73,6 +73,9 @@ nav:
     logs:
       order: 10
       route: 'utils_path'
+    backup:
+      order: 20
+      route: 'backup_path'
```

Comment:
I think we should not offer this feature in the utils menu in Tex. It's meant for creating dump that can only be used for upgrading to Master. The dump cannot be restored on Cloud 5, right? If that's true, offering it in the "Utils" menu is creating wrong expectations I think.
The backup for upgrade should be triggerd with the upgrade wizard (whenever that is implemented)